regulations/static/regulations/css/less/module/full-footer.less

```diff
@@ -0,0 +1,2 @@
+.reg-footer {
```

Comment:
Sure. I know what you're doing because I have the context of other PRs :) The goal is to make this understandable next week.